docs/design_docs/template.md

```diff
@@ -1,21 +1,17 @@
-# ADR {ADR-NUMBER}: {TITLE}
-
-## Changelog
-
-- {Month} {Day}, {Year}: {Changelog} {@User}
+# Design Doc {NUMBER}: {TITLE}
 
 ## Status
 
-> A decision may be "proposed" if it hasn't been agreed upon yet, or "accepted"
-> once it is agreed upon. Once the ADR has been implemented mark the ADR as
-> "implemented". If a later ADR changes or reverses a decision, it may be marked
-> as "deprecated" or "superseded" with a reference to its replacement.
+> A Status may be Draft if we want to break discussion in few PRs to break down a longer process in few steps.
+> At the end of the process each task has to be either Accepted or Declined.
+> If a later task significantly changes or reverses a decision, it may be marked
+> as superseded" with a reference to its replacement.
```

Comment:
```suggestion
> At the end of the process each design has to be either Accepted or Declined.
> If a later design doc significantly changes or reverses a decision, it may be marked
> as superseded" with a reference to its replacement.
```